### PR TITLE
2015 04 cc tabs ui update

### PR DESCRIPTION
**Pull request by local-girl**
Bit of a messy branch, I rebased it a bit - best to look at the overview rather than single commits as design changes came in while I was working on it!

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Tabs/Tabs.ts [hunk 1]
```diff
@@ -78,7 +78,18 @@ export var tabsetDirective = (adhConfig : AdhConfig.IService) => {
...
+            if (attrs.fullWidth) {
+                scope.$watch(() => scope.tabs.length, (value : number) => {
+                    if (value !== 0) {
+                        var tabWidth = Math.floor(100 / value);
+                        element.find(".tabset").addClass("full-width")
+                            .find(".tab").css("width", tabWidth + "%");
+                    }
+                });
+            }
+        }
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Tabs/Tabs.ts]: There should either be a link function or a controller, not both.

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_tabs.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.tabset {
+    &.full-width {
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_tabs.scss]: should be a modifier

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_tabs.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        .tab {
+            text-align: center;
+        }
+    }
+
+    &.pane-above {
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_tabs.scss]: should be a modifier

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_tabs.scss]: This should probably have been removed in 2054947

**Comment by xi** [comment]
If no tab is selected the pane is not hidden completely.

**Comment by xi** [comment]
LGTM. @local-girl could you please review my changes?

**Comment by local-girl** [comment]
LGTM

**Pull request merged**